desktop/cmp/treemap/TreeMap.scss

```diff
@@ -0,0 +1,48 @@
+.xh-treemap {
+  text {
+    font-family: var(--xh-font-family) !important;
+    font-size: var(--xh-font-size-small-px) !important;
```

Comment:
Any (easy-ish) way to override with a more specific selector and avoid !important?  Thinking it might be something an app would want to override.

Another option is to go ahead and setup the CSS vars for these labels.  We might want to do a pass though that takes the chart into consideration as well, and I'm fine if we're not really ready for that.